neurodamus/core/random.py

```diff
@@ -3,57 +3,36 @@
 """
 from __future__ import absolute_import
 from . import Neuron
-from neurodamus.core.configuration import RNGConfig
 
 
-# NOTE: These are pure wrappers, in the sense they dont create Python objects. Instead
+# NOTE: These are pure wrappers, in the sense they don't create Python objects. Instead
 #       Neuron objects are returned (using __new__)
 
 
 class RNG(object):
     def __new__(cls, *ids, **kw):
-        """Creates a default RNG, currently based on ACG"""
+        """Creates a default RNG (Random123) """
         seed = kw.get("seed")
-        return Neuron.h.Random() if seed is None else Neuron.h.Random(seed)
+        return Random123(ids[0], ids[1], ids[2], seed)
```

Comment:
A nitpick, shall we also assert len(ids) ?

Reply:
I think I had it the other way around, RNG() should create some default instance of a random generator (I am not sure this is a good idea, but it is used throughout the code, e.g. in `stimuli.py`). We can do it with seed=0 (deterministic) or seed=current time (stochastic), I think deterministic is OK for now.